src/libs/OptionsListUtils.js

```diff
@@ -54,14 +54,23 @@ Onyx.connect({
     },
 });
 
+const expenseReports = {};
 const iouReports = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT,
-    callback: (iouReport, key) => {
-        if (!iouReport || !key || !iouReport.ownerEmail || !ReportUtils.isIOUReport(iouReport)) {
+    callback: (report, key) => {
+        if (!report || !key || !report.ownerEmail) {
             return;
         }
-        iouReports[key] = iouReport;
+
+        if (ReportUtils.isExpenseReport(report)) {
+            expenseReports[key] = report;
+            return;
```

Comment:
NA(huge)B - I don't think we need this return here, since the helper functions can never both return true. But maybe the linter was going to be mad at you?

Reply:
yeah, I feel like this is kind of just a code style thing. You're right that it's not needed, but personally I prefer having the return because it makes it clear(er) that both things can't be true. I don't think it matters much either way, so I'm just going to resolve this

Reply:
Fair point, but I think readability is a good reason to leave as is here.